test/e2e/tests/helper.py

```diff
@@ -0,0 +1,103 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Helper functions for ec2 tests
+"""
+
+class Ec2Validator:
+    def __init__(self, ec2_client):
+        self.ec2_client = ec2_client
+
+    def assert_dhcp_options(self, dhcp_options_id: str, exists=True):
```

Comment:
This is asserting the existence of DHCP Options
```suggestion
    def assert_dhcp_options_exists(self, dhcp_options_id: str, exists=True):
```

Reply:
Can you clarify? I purposefully omitted "exists" in the func name after looking at APIGateway tests so I could use the same func for both assertions

Reply:
An assertion checks one thing against another. "Assert DHCP Options" doesn't say what it's asserting against, so it's an incomplete statement. "Assert DHCP Options Exists" tells you what it's actually asserting, so that's why it's preferable. It should be updated in the APIGateway tests too, if that's where you got it :)

Reply:
If it sounds incomplete, then I can explicitly pass in the `exists` parameter when checking for existence:
```
# check existence
assert_dhcp_options(id, exists=True)

# check doesn't exist
assert_dhcp_options(id, exists=False)
```

[`APIGateway`](https://github.com/aws-controllers-k8s/apigatewayv2-controller/blob/main/test/e2e/tests/helper.py#L119) has 2 separate funcs for this, `assert_res__is_present`/`asset_res_is_deleted` (maybe this should be renamed for accuracy).

IMO 1 func is enough and clearer, but if we're trying to align styles across controllers to make maintaining easier, then I'm all for it. Details should go in an issue.